WordPress/src/main/java/org/wordpress/android/ui/photopicker/mediapicker/MediaPickerFragment.kt

```diff
@@ -0,0 +1,456 @@
+package org.wordpress.android.ui.photopicker.mediapicker
+
+import android.Manifest.permission
+import android.net.Uri
+import android.os.Bundle
+import android.os.Parcelable
+import android.text.Html
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.PopupMenu
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.GridLayoutManager
+import kotlinx.android.synthetic.main.photo_picker_fragment.*
+import kotlinx.android.synthetic.main.photo_picker_fragment.view.*
+import kotlinx.android.synthetic.main.stats_list_fragment.*
```

Comment:
NP: AS says this import is unused. Wonder why the linter didn't say anything, maybe because the class is not still used 🤔